core/commands/files/files.go

```diff
@@ -756,6 +794,69 @@ are run with the '--flush=false'.
 	},
 }
 
+var FilesUpdateCmd = &cmds.Command{
```

Comment:
not yet sold on `update` as the command name, but its an okay placeholder for now.